.github/workflows/tests.yml

```diff
@@ -7,7 +7,10 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [3.8, 3.7, 2.7]
-        plone-version: [5.2]
+        plone-version: ["6.0", 5.2]
+        exclude:
+          - python-version: 2.7
+            plone-version: 6.0
```

Comment:
Plone 6 supports Python 3.8, 3.9, 3.10, 3.11. It's probably okay to not test all of them here, since we also have the Plone jenkins jobs. But there's no need to test 6.0 on Python 3.7.

Reply:
I am trying to go in small steps here. First, add support for Plone 6, then add Python versions. Otherwise, I have to fix multiple CI failures at the same time.

Reply:
Though, fixing a failure on Python 3.7 and Plone 6 indeed does not make much sense. Thanks for the review! :)